Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Cursors/FingerCursor.cs

```diff
@@ -171,14 +171,18 @@ protected virtual bool IsNearGrabbableObject(out float dist)
         /// <returns></returns>
         protected bool TryGetJoint(TrackedHandJoint joint, out Vector3 position, out Quaternion rotation)
         {
-            if (Pointer != null && Pointer.Controller != null)
+            if (Pointer != null)
             {
-                if (HandJointUtils.TryGetJointPose(joint, Pointer.Controller.ControllerHandedness, out MixedRealityPose handJoint))
+                var hand = Pointer.Controller as IMixedRealityHand;
```

Comment:
i could see this being common code (wanting to get the position of a hand from the pointer or controller). Should we just expose a method HandJointUtils.TryGetJoint(Controller) and HandJointUtils.TryGetJoint(Pointer) ?

Reply:
Yea I wrestled with that. It just seems so easy to accidentally select one of the HandJointUtils options that are way less efficient. I mean that was the original bug here.